### DISCUSSION TITLE
Git tag names

### DISCUSSION THREAD

**Discussion by nim65s** [tiangolo/poetry-version-plugin#57] [Questions] [answered]
Hello,

In my current use cases, in don't have a `1.2.3` tag for the version 1.2.3, but a `v1.2.3` one.

Maybe we could add a little configuration for this in the pyproject.toml file ?

**Comment by juhap** [comment, marked as answer]
I made a quick test and looks like this is already supported. Tried tagging the repository with v0.1.4 and got package with version 0.1.4.
